priv/emqx.schema

```diff
@@ -1607,16 +1596,23 @@ end}.
   {datatype, {duration, ms}}
 ]}.
 
-{mapping, "bridge.$name.retry_interval", "emqx.bridges", [
-  {default, "20s"},
-  {datatype, {duration, ms}}
-]}.
-
```

Comment:
Why did you delete this config entry ?

Reply:
there is no more delay-retry for sending messages, rather a delay-retry of reconnect, messages to retry will be sent right after a successful reconnect.

Reply:
see `bridge.$name.reconnect_interval`